### PR TITLE
Issue/4527 dashboard sections charts

### LINKED ISSUE #4527: [DASHBOARD] 1.3 Create the different sections on the page
Create the different sections on the page, based on the navigation bar structure. 

TODO: add design for it.

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

* Short description here *

closes #4527 #4528 #4529 #4530 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [ ] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: changelogs/unreleased/4527-dashboard-sections-charts.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+description: Create Section, GraphCard, LineCharts and Area charts components and display them on Dashboard
+issue-nr: 4527
+change-type: path
```
Comment (LauraStordeur) [inline changelogs/unreleased/4527-dashboard-sections-charts.yml:3]: patch

File: src/Slices/Dashboard/UI/Charts/LineChart.tsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+        )}
+        <ChartGroup>
+          {metrics.map(({ name, data }, metricIndex) => (
+            <ChartScatter
+              data={data.map((value, index) => {
+                return {
+                  x: timestamps[index],
+                  y:
+                    value === null
+                      ? null
```
Comment (LauraStordeur) [inline src/Slices/Dashboard/UI/Charts/LineChart.tsx:205]: Dunno if it's the same double ternary as in the other PR, but same comment anyway. This is not clean ;)

File: src/Slices/Dashboard/UI/helper.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        if (max < tempMax) {
+          max = tempMax;
+        }
+      });
+    }
+  } else {
+    tempCharState = [metrics as Metric];
+    max = (metrics as Metric).data
+      .flat()
+      .map((value) => (value !== null ? value : 0))
```
Comment (LauraStordeur) [inline src/Slices/Dashboard/UI/helper.ts:69]: why not flatmap() instead?

Comment (matborowczyk) [reply src/Slices/Dashboard/UI/helper.ts:69]: I didn't know that one :D

Comment (LauraStordeur) [reply src/Slices/Dashboard/UI/helper.ts:69]: it's a combo move 😄

**Review by LauraStordeur — APPROVED** [review]

**Pull request closed without merging**